src/test/java/edu/harvard/iq/dataverse/DataCitationTest.java

```diff
@@ -266,7 +266,7 @@ public void testToEndNoteString_withTitleAndAuthor() throws ParseException {
             "<edition>V1</edition>" +
             "<publisher>LibraScholar</publisher>" +
             "<urls><related-urls><url>https://doi.org/10.5072/FK2/LK0D1H</url></related-urls></urls>" +
-            "<electronic-resource-num>doi/10.5072/FK2/LK0D1H</electronic-resource-num>" +
+            "<electronic-resource-num>doi:10.5072/FK2/LK0D1H</electronic-resource-num>" +
```

Comment:
It's weird that this old test expected `doi/10...` instead of `doi:10...`. It feels like this PR is cleaning things up! ❤️

Reply:
@qqmyers commented above that this slash might be required by EndNote, although it does look weird to me :thinking: